src/sentry/south_migrations/0476_truncate_symcaches_and_cficaches.py

```diff
@@ -0,0 +1,1328 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+from sentry.models import File, ProjectCfiCacheFile, ProjectSymCacheFile
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = True
+
+    def forwards(self, orm):
+        try:
+            last_symcache_id = ProjectSymCacheFile.objects \
+                .values_list('cache_file_id') \
+                .latest('id')[0]
+        except ProjectSymCacheFile.DoesNotExit:
+            last_symcache_id = None
+
+        try:
+            last_cficache_id = ProjectCfiCacheFile.objects \
+                .values_list('cache_file_id') \
+                .latest('id')[0]
+        except ProjectCfiCacheFile.DoesNotExit:
+            last_symcache_id = None
+
+        if last_symcache_id is not None:
+            ProjectSymCacheFile.objects.all().delete()
+        if last_cficache_id is not None:
+            ProjectCfiCacheFile.objects.all().delete()
+
+        if last_symcache_id is not None:
+            File.objects\
```

Comment:
Same here. These are potentially painful queries to do just at once. How many rows are we talking about here? We also don't have an index on this, so this seems like a really bad query.